cluster/ci/monitoring/mixins/grafana_dashboards/e2e_template_jobs.jsonnet

```diff
@@ -0,0 +1,112 @@
+local grafana = import 'grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local graphPanel = grafana.graphPanel;
+local prometheus = grafana.prometheus;
+local template = grafana.template;
+
+local dashboardConfig = {
+  uid: '68542739-4831-43e0-b5a0-5117369c83c8',
+};
+
+dashboard.new(
+  'e2e template jobs dashboard',
+  time_from='now-12h',
+  schemaVersion=18,
+)
+
+.addPanel(
+  (graphPanel.new(
+     'CPU usage / requested',
+     description='Percentage of CPU requested usage',
+     datasource='prometheus-k8s',
+     legend_alignAsTable=true,
+     legend_rightSide=true,
+     legend_values=true,
+     legend_min=true,
+     legend_max=true,
+     legend_current=true,
+   ))
+  .addTarget(prometheus.target(
+    '100 * (sum by(pod)(label_join(pod_name:container_cpu_usage:sum, "pod", "", "pod_name")) * on(pod) group_right(label_component) label_replace(kube_pod_labels {namespace=~"ci-op-.*", label_ci_openshift_io_refs_org=~"$org", label_ci_openshift_io_refs_repo=~"$repos"}, "pod", "$1", "pod", "(.*)")) / on (pod) group_left sum(label_replace(kube_pod_container_resource_requests_cpu_cores, "pod", "$1", "pod", "(.*)")) by (pod)',
+    legendFormat='{{label_ci_openshift_io_refs_org}}/{{label_ci_openshift_io_refs_repo}}--{{pod}}',
+  ))
+  .resetYaxes()
```

Comment:
Why?

Reply:
I don't have any idea why it is not working without it.

Reply:
I guess notworking because you also did `addYaxis`.
If you want to define the format of `Y`, maybe do this?
`formatY1='percentunit', `

eg, 

https://github.com/openshift/release/blob/f776acc06ab821e2e842bf530793a9a39a9301ee/cluster/ci/monitoring/mixins/grafana_dashboards/hook.jsonnet#L87-L98

Reply:
addressed.